packages-next/admin-ui/src/components/Navigation.tsx

```diff
@@ -86,6 +83,26 @@ const SignoutButton: FunctionComponent = () => {
   );
 };
 
+const AuthenticatedItem: FunctionComponent<{
+  item: { id: string; label: string };
+}> = ({ item }) => {
```

Comment:
non-blocking: pls never use the `FunctionComponent` type

Reply:
👍 Noted

I copied this from existing usage, will do a blitz on them